opensearch-observability/src/main/kotlin/org/opensearch/observability/model/OperationalPanel.kt

```diff
@@ -0,0 +1,501 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+package org.opensearch.observability.model
+
+import org.opensearch.common.io.stream.StreamInput
+import org.opensearch.common.io.stream.StreamOutput
+import org.opensearch.common.io.stream.Writeable
+import org.opensearch.common.xcontent.ToXContent
+import org.opensearch.common.xcontent.XContentBuilder
+import org.opensearch.common.xcontent.XContentFactory
+import org.opensearch.common.xcontent.XContentParser
+import org.opensearch.common.xcontent.XContentParserUtils
+import org.opensearch.observability.ObservabilityPlugin.Companion.LOG_PREFIX
+import org.opensearch.observability.util.fieldIfNotNull
+import org.opensearch.observability.util.logger
+
+/**
+ * OperationalPanel main data class.
+ *  * <pre> JSON format
+ * {@code
+ * {
+ *   "operationalPanel": {
+ *     "name": "Demo Panel 1",
+ *     "dateCreated": "2021-07-19T21:01:14.871Z",
+ *     "dateModified": "2021-07-19T21:01:14.871Z",
```

Comment:
Don't need these dates anymore, as all objects use document dates

Reply:
I'll send another PR for this, there are also other changes to data modal